pymc/tests/test_sampling_jax.py

```diff
@@ -44,6 +44,23 @@ def test_transform_samples():
     assert 1.5 < trace.posterior["sigma"].mean() < 2.5
 
 
+def test_deterministic_samples():
+    aesara.config.on_opt_error = "raise"
+    np.random.seed(13244)
+
+    obs = np.random.normal(10, 2, size=100)
+    obs_at = aesara.shared(obs, borrow=True, name="obs")
+    with pm.Model() as model:
+        a = pm.Normal("a", 0, 1)
+        b = pm.Deterministic("b", a / 2.0)
```

Comment:
Let's (also) add a deterministic of a transformed variable? That's often where we find problems.

Reply:
By transform, I mean a variable like `s = pm.Lognormal()`, which has a `logTransform` automatically under the hood. I am not referring to the `pm.Deterministic` transform

Reply:
Sorry I forgot that for this test I draw `a` from a Normal and not a Uniform which has a transform under the hood. I'll add something here.

Reply:
No problem! The test looks great now